mathics/builtin/graphics.py

```diff
@@ -1087,6 +1093,75 @@ def stylebox_style(style, specs):
                     raise BoxExpressionError
             return new_style
 
+        failed = []
+
+        def build_error_box1(style):
+            error_style = style.klass(
+                style.graphics,
+                edge=ERROR_BOX_EDGE_COLOR,
+                face=ERROR_BOX_FACE_COLOR,
+            )
+            if isinstance(self, GraphicsElements):
+                error_primitive_head = Symbol("PolygonBox")
```

Comment:
Let's  define PolygonBox and Polygon3DBox, in `mathics.core.systemsymbols` and move the assignment of the boxes underneath the definitions of  `ERROR_BOX*COLOR`. Thanks.

Reply:
Since the PR in Mathics-django was merged, this is no longer needed.